browser-test/fixtures/echo.json

```diff
@@ -0,0 +1,1460 @@
+{
+  "nested": {
+    "google": {
+      "nested": {
+        "showcase": {
+          "nested": {
+            "v1beta1": {
+              "options": {
+                "go_package": "github.com/googleapis/gapic-showcase/server/genproto",
+                "java_package": "com.google.showcase.v1beta1",
+                "java_multiple_files": true
+              },
+              "nested": {
+                "Echo": {
+                  "options": {
+                    "(google.api.default_host)": "localhost:7469"
+                  },
+                  "methods": {
+                    "Echo": {
```

Comment:
likewise, wondering if this information is duplicated?

Reply:
So this file was generated using the `pbjs` tool with `echo.proto`.  The intention is to eventually, for each client library, check in the JSONs created using `pbjs` from relevant protos, but for now, I don't think this file currently exists in any library.

Reply:
:+1: sounds good.